iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/Replace3TwoWindingsTransformersByThreeWindingsTransformersTest.java

```diff
@@ -0,0 +1,440 @@
+        ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers replaceExpected = new ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers();
+        replaceExpected.apply(expectedNetwork);
+        t2w1 = expectedNetwork.getTwoWindingsTransformer("3WT-Leg1");
+        assertNotNull(t2w1);
+        t2w2 = expectedNetwork.getTwoWindingsTransformer("3WT-Leg2");
+        assertNotNull(t2w2);
+        t2w3 = expectedNetwork.getTwoWindingsTransformer("3WT-Leg3");
+
+        network = createThreeWindingsTransformerNodeBreakerNetwork();
+        ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers replace = new ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers();
+        replace.apply(network);
+    }
+
+    @Test
+    void testApplyChecks1() {
+        network.getTwoWindingsTransformer(t2w1.getId()).setProperty("unknown property", "unknown value");
+        Replace3TwoWindingsTransformersByThreeWindingsTransformers replace = new Replace3TwoWindingsTransformersByThreeWindingsTransformers();
+        assertThrows(PowsyblException.class, () -> replace.apply(network, true, ReportNode.NO_OP),
+                "An unknown property should fail to apply.");
+        assertDoesNotThrow(() -> replace.apply(network, false, ReportNode.NO_OP),
+                "An unknown property should not throw if throwException is false.");
+    }
+
+    @Test
+    void testApplyChecks2() {
+        network.getTwoWindingsTransformer(t2w2.getId()).setProperty("unknown property", "unknown value");
+        Replace3TwoWindingsTransformersByThreeWindingsTransformers replace = new Replace3TwoWindingsTransformersByThreeWindingsTransformers();
+        assertThrows(PowsyblException.class, () -> replace.apply(network, true, ReportNode.NO_OP),
+                "An unknown property should fail to apply.");
+        assertDoesNotThrow(() -> replace.apply(network, false, ReportNode.NO_OP),
+                "An unknown property should not throw if throwException is false.");
+    }
+
+    @Test
+    void testApplyChecks3() {
+        network.getTwoWindingsTransformer(t2w3.getId()).setProperty("unknown property", "unknown value");
+        Replace3TwoWindingsTransformersByThreeWindingsTransformers replace = new Replace3TwoWindingsTransformersByThreeWindingsTransformers();
+        assertThrows(PowsyblException.class, () -> replace.apply(network, true, ReportNode.NO_OP),
+                "An unknown property should fail to apply.");
+        assertDoesNotThrow(() -> replace.apply(network, false, ReportNode.NO_OP),
+                "An unknown property should not throw if throwException is false.");
+    }
+
+    @Test
+    void testGetName() {
+        AbstractNetworkModification networkModification = new Replace3TwoWindingsTransformersByThreeWindingsTransformers();
+        assertEquals("Replace3TwoWindingsTransformersByThreeWindingsTransformers", networkModification.getName());
+    }
+
+    @Test
+    void testHasImpact() {
```

Comment:
Consider for more consistency to name the methods ```XXXTest``` instead

Reply:
Done